posixlib/src/main/scala/scala/scalanative/posix/sys/socket.scala

```diff
@@ -64,7 +64,7 @@ object socket {
     sa_family_t, // ss_family, // ss_family, sa_len is synthesized if needed
     CUnsignedShort, // __opaquePadTo32
     CUnsignedInt, // opaque, __opaquePadTo64
-    CArray[CUnsignedLongLong, _15] // __opaqueAlignStructure to 8 bytes.
+    CArray[CUnsignedLongLong, _31] // __opaqueAlignStructure to 8 bytes.
```

Comment:
I think an Issue should be created to address this change in
a following PR.    Doubling the size of the structure for all
operating systems is not something we should be doing.

Reply:
@LeeTibbert unfortently it is a blcoker for OpenBSD support :(

Reply:
I am suggesting that this change go in as specified, so that the overall OpenBSD PR can be merged.

I am also suggesting that an Issue be created which specifies this change and its implication for
other operating systems. That way, we can make sure that it gets fixed.  

While we do not want to let cruft/badness creep in, sometimes it is easier to fix an
identified issue in detail after the other 999 uncontroversial changes have been merged.

@WojciechMazur may have a "Oh! That is simple, just do this."  solution that can be applied after this PR is merged.